db/migrate/20200328103438_add_attributes_to_users.rb

```diff
@@ -1,6 +1,7 @@
 class AddAttributesToUsers < ActiveRecord::Migration[6.0]
   def change
     add_column :users, :email, :text, null: false
+    add_index :users, :email
```

Comment:
Perfect, I'd only add `unique: true` as well.

Reply:
no probs, will add that too